src/core/core.js

```diff
@@ -189,8 +189,10 @@ export default class Core {
         verticalKey: verticalKey
       });
     }
+    const locationRadius = this._getLocationRadius();
+    // The backend throws an error when locationRadius is 0
```

Comment:
What is the reason for supporting a locationRadius of 0 if the backend doesn't support it? Is it because they are going to support it in the future?

Reply:
A locationRadius of 0 in the SDK means "Don't set a locationRadius". 
(This is the previous behavior as well, with the old logic living in the FilterOptions component instead of core. Previously, we would set a FilterNode with an empty value if the locationRadius was 0. Now, in order to persist the actual value in core, we need to pass this 0 value through.)
In a distance filter, having a radio button selected that has a locationRadius of 0 is slightly different than not having a radio button selected at all. The query param sent to the backend is the same (locationRadius will be unset) but for the former the radio button will be checked, and we will also display it as an applied filter in the AppliedFilters/ResultsHeader components. We don't do this for the latter.

I think my comment is unclear, though, that is it because the backend does not support locationRadius of 0, the SDK has decided to make that mean "unset my locationRadius"

Reply:
ah, got it!